src/pages/iou/IOUCurrencySelection.js

```diff
@@ -163,69 +166,65 @@ class IOUCurrencySelection extends Component {
-                                style={[styles.textInput]}
-                                value={this.state.searchValue}
-                                onChangeText={this.changeSearchValue}
-                                placeholder="Search"
-                                placeholderTextColor={themeColors.placeholderText}
-                            />
-                        </View>
+                <KeyboardAvoidingView>
+                    <HeaderWithCloseButton
+                        title={this.props.translate('iOUCurrencySelection.selectCurrency')}
+                        onCloseButtonPress={() => Navigation.goBack()}
+                    />
+                    <View style={[styles.flex1, styles.w100]}>
                         <View style={[styles.flex1]}>
-                            <SectionList
-                                bounces={false}
-                                indicatorStyle="white"
-                                keyboardShouldPersistTaps="always"
-                                showsVerticalScrollIndicator={false}
-                                sections={this.getSections()}
-                                keyExtractor={option => option.currencyCode}
-                                stickySectionHeadersEnabled={false}
-                                renderItem={({item, key}) => (
-                                    <OptionRow
-                                        key={key}
-                                        mode="compact"
-                                        option={item}
-                                        onSelectRow={() => this.toggleOption(item.currencyCode)}
-                                        isSelected={item.currencyCode === this.state.selectedCurrency.currencyCode}
-                                        showSelectedState
-                                        hideAdditionalOptionStates
-                                    />
-                                )}
-                                renderSectionHeader={({section: {title}}) => (
-                                    <View>
-                                        <Text style={[styles.p5, styles.textMicroBold, styles.colorHeading]}>
-                                            {title}
-                                        </Text>
-                                    </View>
-                                )}
-                            />
+                            <View style={[styles.ph5, styles.pv3]}>
+                                <TextInputWithFocusStyles
+                                    styleFocusIn={[styles.textInputReversedFocus]}
+                                    ref={el => this.textInput = el}
+                                    style={[styles.textInput]}
+                                    value={this.state.searchValue}
+                                    onChangeText={this.changeSearchValue}
+                                    placeholder="Search"
```

Comment:
Shouldn't this get translated too ?

Reply:
Yeah, it should be. But It does not come under the scope of this PR. Anyways happy to do this change. Let me know, if I should do this change as well.

Reply:
Could you please add it ? That way we don't forget about it.

Reply:
Pushing a commit right away.

Reply:
@Dal-Papa Code pushed. Thanks.